crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -632,10 +632,10 @@ pub fn derive_from_world(input: TokenStream) -> TokenStream {
                 None => {
                     return syn::Error::new(
                         Span::call_site(),
-                        "No #[from_world] attribute was found on any of this enum's variants.",
+                        "No variant found with the attribute `#[from_world]`",
```

Comment:
Nit: personally I prefer this wording
```suggestion
                        "No variant found with the `#[from_world]` attribute",
```